dts/riscv/rapidsilicon/rapidsi_virgo.dtsi

```diff
@@ -85,18 +85,14 @@
 			reg = <0xA0111000 0xFF>;
 			npins = <16>;
 			#pin-cells = <4>;
-			pin-controller;
-			status = "disabled";
+			status = "okay";
```

Comment:
Why is this okay while others are disabled? Why are there inconsistencies here?

Reply:
I think I did this for testing. This should be reverted to disable.